### ISSUE TITLE
Latest pandas does not have DatetimeTZDtypeType

### ISSUE THREAD

**Issue by The-Fonz** [ynqa/pandavro#14]
There was a [recent change](https://github.com/ynqa/pandavro/commit/1fc7fd3ae8482bb18b008cce90d5b6b0117fdf4f) that added a check for type `pd.core.dtypes.dtypes.DatetimeTZDtypeType`. This does not exist any more in the latest version of Pandas, unfortunately, throwing an error.

```
AttributeError: module 'pandas.core.dtypes.dtypes' has no attribute 'DatetimeTZDtypeType'
```

**Comment by The-Fonz** [comment]
Just saw that this is fixed by #12

**Comment by ynqa** [comment]
Yes, and #12 was merged just now 🎉
